mycroft/skills/skill_manager.py

```diff
@@ -338,7 +342,11 @@ def run(self):
             self.skill_updater.post_manifest()
             self._start_settings_update()
 
+        # wait for initial intents training
+        while not self.initial_load_complete:
```

Comment:
This should probably be refactored to an event we can wait for (with some timeout) instead of an infinite loop